lib/experimental-i18n-theme.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
If I understand this correctly, the idea is that this file is shared between core and cli, so both know the entities to be translated.

- I only mention it because there's a likely change coming, as per https://github.com/WordPress/gutenberg/issues/27295#issuecomment-734279954 so want to be sure that we either 1) we have leeway to update the paths as we need it or 2) we implement 27295 before this one.

- Shouldn't this file have the whole path to the element to be translated? This presumes we only translate things within `settings`, which is true now, but it doesn't need to be.

Reply:
Hi @nosolosw,

> I only mention it because there's a likely change coming, as per #27295 (comment) so want to be sure that we either 1) we have leeway to update the paths as we need it or 2) we implement 27295 before this one.


Yes we can update paths, that's the reason why we are putting them in a file wp-cli can consume, wp-cli will be aware of the path updates.

> Shouldn't this file have the whole path to the element to be translated? This presumes we only translate things within settings, which is true now, but it doesn't need to be.

I updated the code to support non-setting paths.